content/en/blog/_posts/2021-02-27-annotating-k8s-for-humans.md

```diff
@@ -0,0 +1,99 @@
+spec:
+  ports:
+  - name: http
+    port: 80
+    targetPort: 8080
+  selector:
+    app: quote
+```
+
+## A Convention for Annotations
+
+Adopting a common convention for annotations ensures consistency and understandability. Typically, you’ll want to attach the annotation to the service object, as services are the high-level resource that maps most clearly to a team’s responsibility. Namespacing your annotations is also very important. Here is one set of conventions, documented at [a8r.io](https://a8r.io), and reproduced below:
+
+| Annotation                                 | Description                                 |
+| ------------------------------------------ | ------------------------------------------- |
+| `a8r.io/description`                       | Unstructured text description of the service for humans.      |
+| `a8r.io/owner`                             | SSO username (GitHub), email address (linked to GitHub account), or unstructured owner description. |
+| `a8r.io/chat`                              | Slack channel, or link to external chat system |
+| `a8r.io/bugs`                              | Link to external bug tracker |
+| `a8r.io/logs`                              | Link to external log viewer |
+| `a8r.io/documentation`                     | Link to external project documentation |
+| `a8r.io/repository`                        | Link to external VCS repository |
+| `a8r.io/support`                           | Link to external support center |
+| `a8r.io/runbook`                           | Link to external project runbook |
+| `a8r.io/incidents`                         | Link to external incident dashboard |
+| `a8r.io/uptime`                            | Link to external uptime dashboard |
+| `a8r.io/performance`                       | Link to external performance dashboard |
+| `a8r.io/dependencies`                      | Unstructured text describing the service dependencies for humans. |
+
+
+## Visualizing Annotations: Service Catalogs
+
+As the number of microservices and annotations proliferate, using `kubectl describe` can get tedious. Moreover, kubectl describe requires every developer to have some direct access to the Kubernetes cluster. A [service catalog](https://www.getambassador.io/learn/kubernetes-glossary/service-catalog) presents an internal, developer-oriented view of all services. 
+
+Historically, service catalogs have not been a key piece of infrastructure, as applications consisted of a single monolith. Today, with distributed development and microservices the norm, service catalogs are increasingly being adopted as a best practice. Some popular options today include Backstage and Ambassador. [Backstage](https://backstage.io/) is a highly flexible project ideal for organizations that wish to customize every aspect of their service catalog, while [Ambassador](https://www.getambassador.io/products/service-catalog) provides a turnkey, cloud-hosted solution that can be set up quickly.
```

Comment:
You could write:
Historically, service catalogs have not been a key piece of infrastructure, as applications consisted of a single monolith. Today, with distributed development and microservices the norm, service catalogs are increasingly being adopted as a best practice. Some popular options today include [Backstage](https://backstage.io/) and [Ambassador Service Catalog](https://www.getambassador.io/products/service-catalog). Backstage is a highly flexible project ideal for organizations that wish to customize every aspect of their service catalog, while Ambassador Service Catalog provides a turnkey, cloud-hosted solution that can be set up quickly.
```
I'd like to see some text to explain that this concept of a service catalog is different from Kubernetes [Service Catalog](https://svc-cat.io/).

Reply:
Thanks! Your feedback is great. I pushed some changes; please let me know your thoughts.